CONTRIBUTING.md

```diff
@@ -44,6 +44,33 @@ make the code easier to navigate.
 
 *(TODO: Provide some notes on how to prepare a development environment which
 includes the needed dependencies.)*
+### Note
+This project requires libwpewebkit-1.0-dev, libwpe-1.0-dev, libwpebackend-fdo-1.0-dev 
+and GTK4 for the build/run process
+
+After cloning the project from [https://github.com/Igalia/cog] do the following 
+to build and get the project running
+-cd into project
+-create a build directory
+-cd into this directory. This is where your build files will be stored.
+-run cmake -DCOG_PLATFORM_X11=ON -DCOG_PLATFORM_GTK4=ON .. to generate the build files.
+-Then run the make command to generate the binaries
+You just finished building cog. To run it, run COG_MODULEDIR=$PWD/modules ./cog --platform=gtk4 [https://www.igalia.com/] 
```

Comment:
I think if you actually ran this command with the square brackets it wouldn't work? Best to just leave out the square brackets, given it's a quoted command - similarly, code markdown would help here (i.e. surrounding the whole command with backticks).